layer/nrlf/core/constants.py

```diff
@@ -113,6 +113,51 @@ def coding_value(self):
     },
 }
 
+TYPE_ATTRIBUTES = {
+    PointerTypes.MENTAL_HEALTH_PLAN.value: {
+        "display": "Mental health crisis plan",
+    },
+    PointerTypes.EMERGENCY_HEALTHCARE_PLAN.value: {
+        "display": "Emergency health care plan",
+    },
+    PointerTypes.EOL_COORDINATION_SUMMARY.value: {
+        "display": "End of Life Care Coordination Summary",
+    },
```

Comment:
This is slightly different from https://github.com/NHSDigital/NRLF/blob/develop/resources/fhir/NRLF-RecordType-ValueSet.json, since it's case sensitive could this introduce problems? Is there a way to use the actual value set values here without duplicating them all manually?

Reply:
That's a great idea Kate, I fixed the case of the display and as I mention we can do the DRY change in another ticket.